pootle/apps/pootle_store/views.py

```diff
@@ -598,7 +600,12 @@ def submit(request, unit):
             for field, old_value, new_value in form.updated_fields:
                 if field == SubmissionFields.TARGET and suggestion:
                     old_value = str(suggestion.target_f)
-                sub = Submission(
+
+                if current_time.microsecond > 0:
```

Comment:
Why do we need to manipulate with the submission time here?

Reply:
It shouldn't go here. It slipped from an alternate solution where I would add milliseconds support on MySQL. Deleting it